src/Sylius/Component/Attribute/AttributeType/AttributeTypeInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Attribute\AttributeType;
+
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface AttributeTypeInterface
+{
+    const DEFAULT_ATTRIBUTE_TYPE = 'text';
```

Comment:
Should interface even know about the attribute types that exists in the application?